### PR TITLE
Rewrite that actually works

### PR DESCRIPTION
**Pull request by TesDevelopment**
Legit not joking can you actually review this?

### CODE REVIEW

**Review by stangithuboffical — COMMENTED** [review]
doesn't display name

File: src/lua/admin.plugin.lua [hunk 1]
```diff
@@ -1,33 +1,91 @@
...
+                end
+            elseif debug then
+                Utils.Notify(LocalPlayer,"No commands in plugin",string.format("Plugin %s has no commands. Skipping.",plugin.Name),5)
+            end
+        end)
+        if not ss then
+            if debug then
+                math.randomseed(os.time())
+                local a = math.random(999)
+                Utils.Notify(LocalPlayer,"Error loading plugin",string.format("Plugin %s errored on load, saving log as fate_plugincrash_%s.txt",plugin.Name,a),5)
```
Comment (stangithuboffical) [inline src/lua/admin.plugin.lua:68]: this could flood

File: src/lua/admin.plugin.lua [hunk 2]
```diff
@@ -1,33 +1,91 @@
...
+                    Clone.Visible = true
+                    Clone.Parent = Commands.Frame.List
+                end
+            elseif debug then
+                Utils.Notify(LocalPlayer,"No commands in plugin",string.format("Plugin %s has no commands. Skipping.",plugin.Name),5)
+            end
+        end)
+        if not ss then
+            if debug then
+                math.randomseed(os.time())
```
Comment (stangithuboffical) [inline src/lua/admin.plugin.lua:66]: do something like math.randomseed(os.clock()*10000000)

**Review by pozm — COMMENTED** [review]
bad changes again

**Comment by pozm** [comment]
mald more kid literally terrible code.

**Pull request closed without merging**